### DISCUSSION TITLE
mishandling of the edit box

### DISCUSSION THREAD

**Discussion by CodeLazier** [godotengine/godot-proposals#6991] [Editor]
In Godot, a numerical input field can be incremented or decremented by left-clicking and dragging the mouse. However, there is no error calculation implemented, which sometimes results in unintentional value changes when clicking on the input field. It is necessary to implement a trigger for this operation after dragging a certain distance in pixels.

**Comment by CodeLazier** [comment]
![test](https://github.com/godotengine/godot-proposals/assets/14953625/8be6321e-70db-4bb1-bb2a-81723a348867)

**Comment by Calinou** [comment]
Something similar to https://github.com/godotengine/godot/pull/49310 needs to be implemented for SpinBox and EditorSpinSlider. That said, I thought this was done already.
